benchmark/tool/C/README.md

```diff
@@ -7,45 +7,69 @@ The demo can be run from the command line and can be used to test the inference
+            -DCMAKE_BUILD_TYPE=MinSizeRel
 
-    make
+    $ make
     ```
 
-3. Prepare a merged model:   
-Models config(.py) (eg: [Mobilenet](https://github.com/PaddlePaddle/Mobile/blob/develop/models/mobilenet.py)) contain only the structure of our models. A developer can choose [model config here](https://github.com/PaddlePaddle/Mobile/tree/develop/models) to train their custom models. PaddlePaddle documentation has [several tutorials](https://github.com/PaddlePaddle/models) for building and training models. The model parameter file(.tar.gz) will be generated during the training process. There we need to merge the configuration file(.py) and the parameter file(.tar.gz) into a file. Please refer to the [details.](https://github.com/PaddlePaddle/Mobile/tree/develop/tools/merge_config_parameters)
+- **Step 3, prepare a merged model.**
+
+  Models config(.py) (eg: [Mobilenet](https://github.com/PaddlePaddle/Mobile/blob/develop/models/mobilenet.py)) contain only the structure of our models. A developer can choose [model config here](https://github.com/PaddlePaddle/Mobile/tree/develop/models) to train their custom models. PaddlePaddle documentation has [several tutorials](https://github.com/PaddlePaddle/models) for building and training models. The model parameter file(.tar.gz) will be generated during the training process. There we need to merge the configuration file(.py) and the parameter file(.tar.gz) into a file. Please refer to the [details.](https://github.com/PaddlePaddle/Mobile/tree/develop/tools/merge_config_parameters)
 
-4. Run the demo program by logging into the Android environment via [adb](https://developer.android.google.cn/studio/command-line/adb.html?hl=zh-cn#Enabling) and specifying the PaddlePaddle model from the command line as follows:
-	```
-	adb push inference mobilenet.paddle /sdcard/test_mobilenet
-	adb shell
-	cd /sdcard/test_mobilenet
-	./inference --merged_model ./mobilenet.paddle --input_size 150528
-	```
+- **Step 4, run the demo.**
+
+  Users can run the demo program by logging into the Android environment via [adb](https://developer.android.google.cn/studio/command-line/adb.html?hl=zh-cn#Enabling) and specifying the PaddlePaddle model from the command line as follows:
+
+    ```bash
+    $ adb push inference /data/local/tmp # transfer the executable to Android's memory
+    $ adb push mobilenet_flowers102.paddle /data/local/tmp # transfer the model to Android's memory
+    $ adb shell # login Android device
+    odin:/ $ cd /data/local/tmp # switch to the working directory
+    odin:/data/local/tmp $ ls
+    inference  mobilenet_flowers102.paddle
+    odin:/data/local/tmp $ chmod +x inference
+    odin:/data/local/tmp $ ./inference --merged_model ./mobilenet_flowers102.paddle --input_size 150528 # run the executable
+    I1211 17:12:53.334666  4858 Util.cpp:166] commandline:
+    Time of init paddle 3.4388 ms.
+    Time of create from merged model file 141.045 ms.
+    Time of forward time 398.818 ms.
```

Comment:
It is best to indicate this example is running on which phone.

Reply:
I just want to show the output of `inference`, the time here is not precise. I will add a comment about this in next PR.